tests/core/focusmanager/lostfocus.js

```diff
@@ -0,0 +1,34 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: divarea */
+
+bender.editor = true;
+
+bender.test( {
+	'test lost focus in editor after click': function() {
+		// Test for checking if focusmanager reset ranges in divarea editor (bug introduced in https://dev.ckeditor.com/ticket/13446).
+		// It's necessary to keep focus in a browser to get proper result of test. Having focus in a console or other window will cause test to
```

Comment:
This comment is a little confusing:

> Having focus in a console or other window will cause test to pass

Does that mean it will not pass when having focus in a browser? Not sure what is the described behaviour here? Did you mean that test will pass always (even without the fix) when focus is not in the browser?